packages/middleware-render-error-info/lib/render-error-page.js

```diff
@@ -0,0 +1,388 @@
+/**
+ * @module @dotcom-reliability-kit/middleware-render-error-info/lib/render-error-page
+ */
+
+const renderLayout = require('./render-layout');
+
+/**
+ * @typedef {object} ErrorRenderingOptions
+ * @property {import('express').Request} request
+ *     An Express request object.
+ * @property {import('express').Response} response
+ *     An Express response object.
+ * @property {import('@dotcom-reliability-kit/serialize-error').SerializedError} serializedError
+ *     The error to render.
+ */
+
+/**
+ * Render an HTML error info page.
+ *
+ * @access private
+ * @param {ErrorRenderingOptions} options
+ *     Options which impact the rendering of the page.
+ * @returns {string}
+ *     Returns the rendered error page.
+ */
+function renderErrorPage({ request, response, serializedError }) {
+	return renderLayout({
+		body: `
+			<h1 id="errors">Error information</h1>
+			${renderError(serializedError)}
+			${renderRequest(request)}
+			${renderResponse(response)}
+		`,
+		request,
+		response
+	});
+}
+
+/**
+ * Render a serialized error to HTML.
+ *
+ * @access private
+ * @param {import('@dotcom-reliability-kit/serialize-error').SerializedError} error
+ *     The error information to render.
+ * @returns {string}
+ *     Returns the rendered error.
+ */
+function renderError(error) {
+	let warning = '';
+
+	if (!error.stack) {
+		warning = renderWarning({
+			title: 'A non-Error object was thrown.',
```

Comment:
This is cool. Do we want to propose adding [no-throw-literal](https://eslint.org/docs/latest/rules/no-throw-literal) to our standard lint config?

Reply:
Yes let's propose that. We do mention using this and [`prefer-promise-reject-errors`](https://eslint.org/docs/latest/rules/prefer-promise-reject-errors) in the [linked documentation](https://github.com/Financial-Times/dotcom-reliability-kit/blob/main/docs/getting-started/throwing-errors.md#using-error-objects) too